### ISSUE TITLE
NTR: Skin burn, highly-localized (not sunburn) (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8014]
New term request Skin burn, highly-localized (not sunburn) (seidel_190206174842): Epidermis; epidermal  UMLS: 0565944

**Comment by pnrobinson** [comment]
@MickeySegal
what is the intended context? Does this refer to things like a cigarette butt burn?

**Comment by MickeySegal** [comment]
Typically traumatic burns

**Comment by pnrobinson** [comment]
@MickeySegal 
this is difficult in the context of the HPO (to indicate etiology or not-etiology), also highly localized is difficult to objectify across the context of thousands of potential etiologies.
I wonder if it would be better to make a term called
Skin burn
def: Damage to the skin and underlying tissues caused by an agents such as heat, chemicals, electricity, radiation, or friction, resulting in pain, redness, swelling, and sometimes blisters or charring.

**Comment by MickeySegal** [comment]
That is OK, though it would be good to distinguish this from sunburn.

**Comment by pnrobinson** [comment]
@MickeySegal 
added the restriction to "non sunburn" to the comment

Adding new term: 
- ID: HP:6001253
- label: Skin burn
- parents: [Abnormal skin morphology (HP:0011121)](https://hpo.jax.org/app/browse/term/HP:0011121)
- pmid: 
- def: Damage to the skin and underlying tissues caused by an agents such as heat, chemicals, electricity, radiation, or friction, resulting in pain, redness, swelling, and sometimes blisters or charring.
- comment: This term does not refer to sunburn.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8014
- synonyms:
